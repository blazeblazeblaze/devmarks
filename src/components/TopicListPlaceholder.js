import React from "react";
import ContentLoader from "react-content-loader";

const TopicListPlaceholder = props => (
  <section className="section">
    <div className="container">
      <ContentLoader
        speed={2}
        primaryColor="#f3f3f3"
        secondaryColor="#ecebeb"
        {...props}
      >
        <rect x="0" y="0" rx="0" ry="0" width="125" height="30" />
        <rect x="0" y="45" rx="0" ry="0" width="125" height="30" />
        <rect x="140" y="0" rx="0" ry="0" width="125" height="30" />
        <rect x="280" y="0" rx="0" ry="0" width="125" height="30" />
        <rect x="140" y="45" rx="0" ry="0" width="125" height="30" />
        <rect x="280" y="45" rx="0" ry="0" width="125" height="30" />
      </ContentLoader>
    </div>
  </section>
);
export default TopicListPlaceholder;
