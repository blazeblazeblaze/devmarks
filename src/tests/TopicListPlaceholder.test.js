import React from "react";
import TopicListPlaceholder from "../components/TopicListPlaceholder";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

configure({ adapter: new Adapter() });

describe("<TopicListPlaceholder />", () => {
  const tree = shallow(<TopicListPlaceholder />);
  it("matches snapshot", () => {
    expect(toJson(tree)).toMatchSnapshot();
  });
});
