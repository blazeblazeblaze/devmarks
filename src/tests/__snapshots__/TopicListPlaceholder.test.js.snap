// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TopicListPlaceholder /> matches snapshot 1`] = `
<section
  className="section"
>
  <div
    className="container"
  >
    <ContentLoader
      primaryColor="#f3f3f3"
      secondaryColor="#ecebeb"
      speed={2}
    >
      <rect
        height="30"
        rx="0"
        ry="0"
        width="125"
        x="0"
        y="0"
      />
      <rect
        height="30"
        rx="0"
        ry="0"
        width="125"
        x="0"
        y="45"
      />
      <rect
        height="30"
        rx="0"
        ry="0"
        width="125"
        x="140"
        y="0"
      />
      <rect
        height="30"
        rx="0"
        ry="0"
        width="125"
        x="280"
        y="0"
      />
      <rect
        height="30"
        rx="0"
        ry="0"
        width="125"
        x="140"
        y="45"
      />
      <rect
        height="30"
        rx="0"
        ry="0"
        width="125"
        x="280"
        y="45"
      />
    </ContentLoader>
  </div>
</section>
`;
